iotfunctions/db.py

```diff
@@ -1608,7 +1608,7 @@ def register_functions(self, functions, url=None, raise_error=True, force_preins
             payload = {'name': name, 'description': f.__doc__, 'category': category,
                        'moduleAndTargetName': module_and_target, 'url': package_url, 'input': input_list,
                        'output': output_list, 'incremental_update': True if category == 'AGGREGATOR' else None,
-                       'tags': tags}
+                       'tags': tags, 'scope': {'enabled': f.is_scope_enabled}}
```

Comment:
@ValerianClerc 
We'll need to add `scope` for preinstalled functions https://github.com/ibm-watson-iot/functions/pull/229/files#diff-b45e38b37103061720de051c73229a9fR1638.
This will be utilized while installing a new instance or re-initializing an instance of asset monitor.